src/components/CalendarMonth.jsx

```diff
@@ -132,10 +132,13 @@ export default class CalendarMonth extends React.Component {
 
     return (
       <div className={calendarMonthClasses} data-visible={isVisible}>
-        <table>
-          <caption className="CalendarMonth__caption js-CalendarMonth__caption">
-            <strong>{monthTitle}</strong>
-          </caption>
+        <div
+          id="CalendarMonth__caption"
+          className="CalendarMonth__caption js-CalendarMonth__caption"
+        >
+          <strong>{monthTitle}</strong>
+        </div>
```

Comment:
Changed `<caption>` to `<div>` as apparently this is a structural type element that cues screen readers that this is a data vs layout type table (see NOTE on "layout tables" and the caption element here: [H39: Using caption elements to associate data table captions with data tables](https://www.w3.org/TR/WCAG20-TECHS/H39)).

See in DayPicker.jsx that it now points here for the `aria-describedby` and thus continues to read the title as it was before with the `<caption>`.